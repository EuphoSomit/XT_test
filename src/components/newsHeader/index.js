import {NewsHeaderMemo as NewsHeader} from './NewsHeader';
export {NewsHeader as default}