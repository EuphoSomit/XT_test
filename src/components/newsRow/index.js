import {NewsRow} from './NewsRow';
export {NewsRow as default}