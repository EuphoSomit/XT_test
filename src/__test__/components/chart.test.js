import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import {Chart} from './../../components/chart';



test('Chart component snapshot testing', () => {

  let chartData = [
      {"name":"16582136","vote":6015},
      {"name":"11116274","vote":5771},
      {"name":"3078128","vote":4271},
      {"name":"13682022","vote":4107},
      {"name":"23065782","vote":3816},
      {"name":"22107823","vote":3592},
      {"name":"15924794","vote":3384},
      {"name":"3742902","vote":3381},
      {"name":"20052623","vote":3287}
      ];

      console.log("renderer", ReactTestRenderer);

      const component = ReactTestRenderer.create(
        <Chart chartData={ chartData } />
      );
    
    let tree = component.toJSON();
   
      expect(tree).toMatchSnapshot();
})
