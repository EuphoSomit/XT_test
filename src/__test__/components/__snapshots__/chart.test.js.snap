// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart component snapshot testing 1`] = `
<div
  className="chart-container"
/>
`;
